api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -44,6 +45,7 @@
  * PatientService class
  */
 public class ProgramWorkflowServiceTest extends BaseContextSensitiveTest {
+	private ProgramWorkflowDAO dao = null;
```

Comment:
Why are you adding this?

Reply:
This was work of another issue. I've mistakenly put it here. Let me remove it . :)